auto-complete.js

```diff
@@ -195,8 +199,14 @@
                 else
                     that.sc.style.display = 'none';
             }
+            addEvent(that, 'click', function (e) {
+                if (o.queryHistoryStorageName) {
+                    o.source(that.value, suggest);
```

Comment:
This will call the `source` function that calls the autocomplete API, this is not necessary to show local suggestions.
https://github.com/jusbrasil/ui-topbar/blob/8870d9c4357063f06a628ca7ac5b2813aacba719/src/tags/autocomplete/jusbrasil.js#L93

Create a function for this purpose only.

Reply:
The function ```source``` is actually called because before, if the user clicked outside the text box and clicked again, the suggestions were not shown.
With this now, if the user clicks outside the text box and clicks again, the suggestions will be displayed, being from local storage or not!